### PR TITLE
Blog Post: 2024-03-14

### PR DESCRIPTION
**Pull request by oandrefonseca**
Blog Post: 2024-03-14

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,6 @@
 _site/
 /.quarto/
+.gitignore
```
Comment (diegomscoelho) [inline .gitignore:3]: Need to remove this.

**Review by diegomscoelho — COMMENTED** [review]
Just a first look. Going to test it.

**Comment by diegomscoelho** [comment]
Changed PR name to `Blog Post: 2024-03-14`. @iaradsouza1 created this pattern.

File: .gitignore (hunk 1 above)
Comment (oandrefonseca) [reply .gitignore:3]: Ops... My bad... What should I do? Remove the PR and push it again?

Comment (diegomscoelho) [reply .gitignore:3]: Just need to accept suggestion. Sorry to not include before.
```suggestion
```

File: posts/2024-03-14/index.qmd [hunk 2]
```diff
@@ -0,0 +1,90 @@
+---
+title: "Data interoperability for single-cell analysis"
+author: "Lexanomics"
+date: "2024-03-14"
+execute:
+    warning: false
+    freeze: auto
+categories: [debbuging new]
```
Comment (diegomscoelho) [inline posts/2024-03-14/index.qmd:8]: Juicy
```suggestion
categories: [code, debbuging new]
```

Comment (iaradsouza1) [reply posts/2024-03-14/index.qmd:8]: I really dont like the idea of several tags, I think fewer are better

**Review by iaradsouza1 — COMMENTED** [review]
Ready to go

**Review by diegomscoelho — APPROVED** [review]
Lgtm

**Pull request merged**